tensorflow/python/training/learning_rate_decay_test.py

```diff
@@ -23,8 +23,11 @@
 from tensorflow.python.ops import state_ops
 from tensorflow.python.ops import variables
 from tensorflow.python.platform import googletest
+from tensorflow.python.training import initial_learning_rate_decay
 from tensorflow.python.training import learning_rate_decay
 
+import math
```

Comment:
we typically import system / python libraries before tensorflow ones, so you can add this on line 20 like:

    from __future__ import print_function

    import math

    from tensorflow.python. ....